regex.c

```diff
@@ -86,6 +95,10 @@ bool match_regex(char *input)
 		{
 			// Match, return true
 			matched = true;
+
+			// Print match message when in regex debug mode
+			if(config.regex_debugmode)
+				logg("DEBUG: Regex in line %i (\"%s\") matches \"%s\"", index+1, regexbuffer[index], input);
```

Comment:
It might make it clearer if we only wrap the regex in parentheses, and not parentheses and quotes.